### PR TITLE
Issue #1175 Clarify required privilege elevation in docs

### PR DESCRIPTION
**Pull request by robin-owen**
**Fixes:** Issue #1175

### CODE REVIEW

**Comment by robin-owen** [comment]
Note that I'm unsure about the required permissions for `crc start` on Linux/macOS -- is the ability to elevate privileges using `sudo` required? If so, I can update this PR with that information as well.

**Comment by anjannath** [comment]
> Note that I'm unsure about the required permissions for crc start on Linux/macOS -- is the ability to elevate privileges using sudo required?

Not needed for linux, but for macOS during `crc start` a network restart might be needed and to perform that sudo is used.

**Comment by praveenkumar** [comment]
> Not needed for linux, but for macOS during crc start a network restart might be needed and to perform that sudo is used.

No on the mac also we don't need any sudo operation as part of `crc start` the network restart happen using logged in user.

File: docs/source/topics/proc_starting-the-virtual-machine.adoc [hunk 1]
```diff
@@ -8,6 +8,7 @@ The [command]`{bin} start` command starts the {prod} virtual machine and OpenShi
 * To avoid networking-related issues, ensure that you are not connected to a VPN and that your network connection is reliable.
 * You set up the host machine through the [command]`{bin} setup` command.
 For more information, see link:{crc-gsg-url}#setting-up-codeready-containers_gsg[Setting up {prod}].
+* On {msw}, ensure that your user account can elevate to Administrator privileges.
```
Comment (praveenkumar) [inline docs/source/topics/proc_starting-the-virtual-machine.adoc:11]: :+1:

**Pull request merged**